editor/resources/editor.fxml

```diff
@@ -9,237 +9,242 @@
-                        <TitledPane animated="false" collapsible="false" text="Call Stack" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
-                          <ListView id="debugger-call-stack"/>
-                        </TitledPane>
-                      </AnchorPane>
-                      <AnchorPane prefHeight="100.0" prefWidth="160.0">
-                        <TitledPane animated="false" collapsible="false" text="Variables" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
-                          <TreeView id="debugger-variables"/>
-                        </TitledPane>
-                      </AnchorPane>
-                    </SplitPane>
-                  </children>
-                </AnchorPane>
-              </items>
-            </SplitPane>
-          </children>
-        </AnchorPane>
-      </items>
-    </SplitPane>
-      <HBox id="status-bar">
-        <children>
-          <Label id="status-label" text="Ready" HBox.hgrow="ALWAYS" maxWidth="1.7976931348623157E308"/>
-          <StackPane id="status-pane">
-            <Hyperlink id="update-link" text="Update Available" visible="false" StackPane.alignment="CENTER_RIGHT"/>
-            <HBox id="progress-hbox" visible="false">
-              <Label id="progress-percentage-label"/>
-              <ProgressBar id="progress-bar" styleClass="really-small-progress-bar"/>
-              <Button id="progress-cancel-button" mnemonicParsing="false" visible="false">
-                <graphic>
-                  <Region id="cross"/>
-                </graphic>
-              </Button>
-            </HBox>
-          </StackPane>
-        </children>
-      </HBox>
+                      </SplitPane>
+                    </children>
+                  </AnchorPane>
+                </items>
+              </SplitPane>
+            </children>
+          </AnchorPane>
+        </items>
+      </SplitPane>
+      <VBox id="notifications" maxWidth="-Infinity" maxHeight="-Infinity" StackPane.alignment="BOTTOM_RIGHT" pickOnBounds="false">
+      </VBox>
+    </StackPane>
+    <HBox id="status-bar">
+      <children>
+        <Label id="status-label" text="Ready" HBox.hgrow="ALWAYS" maxWidth="1.7976931348623157E308" />
+        <StackPane id="status-pane">
+          <Hyperlink id="update-link" text="Update Available" visible="false" StackPane.alignment="CENTER_RIGHT" />
+          <HBox id="progress-hbox" visible="false">
+            <Label id="progress-percentage-label" />
+            <ProgressBar id="progress-bar" styleClass="really-small-progress-bar" />
+            <Button id="progress-cancel-button" mnemonicParsing="false" visible="false">
+              <graphic>
+                <Region id="cross" />
+              </graphic>
+            </Button>
+          </HBox>
+        </StackPane>
+      </children>
+    </HBox>
```

Comment:
Very big diff here, though it's mostly indentation. The changes here are:
1. wrap the `#main-split` SplitPane in a StackPane
2. add `#notifications` VBox in the StackPane for notifications that are shown on top of the main split.